graphs/03-graphs-algorithms.md

```diff
@@ -596,14 +621,22 @@ In the worst-case you will need to add each node to the Stack, so the space comp
 ### !end-challenge
 
 <!-- ======================= END CHALLENGE ======================= -->
+## Choosing Between Traversal Algorithms
+
+How do we decide which algorithm to use? Oftentimes, problems requiring graph traversal can be solved with either breadth first search or depth first search.
+
+If we expect the solution to be relatively close to our start node, we may choose to breadth first search because it explores nodes closest to it first. Conversely, if we expect the solution to be much further away from the start node, we may choose to use depth first search. 
```

Comment:
```suggestion
If we expect the solution to be relatively close to our start node, we may choose to use breadth first search because it explores nodes closest to it first. Conversely, if we expect the solution to be much further away from the start node, we may choose to use depth first search. 
```